spec/models/time_based_repo_rater_spec.rb

```diff
@@ -9,125 +9,125 @@
       let(:repo) { nil }
 
       it 'should rate minimum value' do
-        expect(subject.rate(repo)).to eq(0)
+        expect(subject.rate).to eq(0)
       end
     end
 
     context 'repo with invalid date' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', 'invalid date', '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', 'invalid date', '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'invalid-repo')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'invalid-repo')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate minimum value' do
-        expect(subject.rate(repo)).to eq(0)
+        expect(subject.rate).to eq(0)
       end
 
     end
 
     context 'repo with invalid date' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', nil, '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', nil, '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'invalid-repo')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'invalid-repo')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate minimum value' do
-        expect(subject.rate(repo)).to eq(0)
+        expect(subject.rate).to eq(0)
       end
     end
 
     context 'repo with commit within 1 day' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', Time.now, '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', Time.zone.now, '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'repo-name')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'repo-name')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate maximum value' do
-        expect(subject.rate(repo)).to eq(10)
+        expect(subject.rate).to eq(10)
       end
     end
 
     context 'repo with commit within 1 week' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', time_days_ago(7), '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', time_days_ago(7), '') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.